imap_processing/codice/constants.py

```diff
@@ -1023,6 +1014,529 @@
     255: 4294967294,
 }
 
```

Comment:
Can you add a link for where this data/table comes from and how it is organized? Your comment on the variable below this is a great example!